python/hsfs/core/feature_view_engine.py

```diff
@@ -119,8 +119,8 @@ def get_batch_query_string(
         query_obj = self._feature_view_api.get_batch_query(
             feature_view_obj.name,
             feature_view_obj.version,
-            start_time,
-            end_time,
+            util.convert_event_time_to_timestamp(start_time),
```

Comment:
Why is the inpur converted in `get_batch_query_string`, but not in `get_batch_query`?

Reply:
get_batch_query calls get_batch_query_string